ArduCopter/Copter.cpp

```diff
@@ -202,6 +202,7 @@ const AP_Scheduler::Task Copter::scheduler_tasks[] = {
 #if STATS_ENABLED == ENABLED
     SCHED_TASK_CLASS(AP_Stats,             &copter.g2.stats,            update,           1, 100),
 #endif
+    SCHED_TASK(send_telemetry_loop_400Hz,400,    90),
```

Comment:
I think rather than being copter only this should be in the AP_Vehcile sheduler table, https://github.com/ArduPilot/ardupilot/blob/164da261f10f0f6559330cf1d2dba875dd757eea/libraries/AP_Vehicle/AP_Vehicle.cpp#L193 then we get it on all vehicles. You will have to move the init stuff to vehicle also. You can use the `SCHED_TASK_CLASS` to go direct rather than needing the `send_telemetry_loop_400Hz` helper